### ISSUE TITLE
Val MVP: Edit Hyrax::FileSet as a valkyrie resource through UI

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#5137]
### Descriptive summary

As part of the Valkyrization MVP, a user should be able to edit file sets through the UI with the underlying code processing the file set as a `Hyrax:: FileSet`.

### Rationale

Part of the basic Valkyrization MVP for testing the progress of the valkyrization effort.

### Steps to reproduce the behavior

1. Login to [nurax-pg](https://nurax-pg.curationexperts.com)
1. Dashboard -> [Works](https://nurax-pg.curationexperts.com/dashboard/my/works?locale=en)
2. Click the title of an existing work that has a file
2. In the file listing, click the title of a file
3. On the File Set show page, click Edit This File
4. Change the title
4. Save

The file set should be updated and the show page for the file set should display with the updated title.

### Related work

See all work tagged with Milestone [Valkyrization MVP](https://github.com/samvera/hyrax/milestone/19)

**Comment by MPLSFedResearchTZ** [comment]
The same problem as in https://github.com/samvera/hyrax/issues/5135#issuecomment-965648961 . Could not added files to a work.

**Comment by no-reply** [comment]
@orangewolf this got moved from backlog to ready for review last week and we're wondering what work should be the next step here? is there a PR that needs attention? @dlpierce was wondering whether there's something that still needs to be deployed to nurax-pg?

**Comment by dunn** [comment]
This does currently work in koppie, fwiw

**Comment by orangewolf** [comment]
this is one of those items that was blocked by something before it being broken (couldn't add files to a work so you sure as heck couldn't edit those files). We fixed the add files and I did test editing files as part of that work not even knowing about this ticket. This ticket was later discovered and so it got moved to needed testing live in some meeting or another. I believe this functionality should work on both koppie (thanks @dunn ) and nurax-pg at this point.
